### PR TITLE
4.27 release change in SDK4Mvn.aggr

### PR DESCRIPTION
**Pull request by elsazac**
changed the version to 4.27 and updated with latest build id.

### CODE REVIEW

File: publish-to-maven-central/SDK4Mvn.aggr [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>4.27-I202303111800/Users/elsazacharia/releng/eclipse.platform.releng :wq:
```
Comment (akurtakov) [inline publish-to-maven-central/SDK4Mvn.aggr:1]: This is wrong.

File: publish-to-maven-central/SDK4Mvn.aggr [hunk 2]
```diff
@@ -37,4 +37,4 @@
   <mavenDependencyMapping namespacePattern="osgi\.bundle|java\.package" namePattern="org\.apache\.jasper\..*" groupId="*" artifactId="*" versionRangePattern=".*" versionRangeTemplate="major.minor.micro.v201112011158"/>
   <mavenDependencyMapping namespacePattern="osgi\.bundle|org\.eclipse\.equinox\.p2\.iu" namePattern="(?!.*(org\.eclipse\.(emf|ecf))).*" groupId="*" artifactId="*" versionRangePattern=".*" versionRangeTemplate="major.minor.micro-qualifier"/>
   <mavenDependencyMapping namespacePattern="java\.package" namePattern=".*" groupId="*" artifactId="*" versionRangePattern=".*" versionRangeTemplate="major.minor.micro-qualifier"/>
-</aggregator:Aggregation>
+</aggregator:Aggregation>4.27-I202303111804.27-I2023031118000
```
Comment (akurtakov) [inline publish-to-maven-central/SDK4Mvn.aggr:40]: And so is this.

File: publish-to-maven-central/SDK4Mvn.aggr [hunk 3]
```diff
@@ -2,10 +2,10 @@
 <aggregator:Aggregation xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:aggregator="http://www.eclipse.org/cbi/p2repo/2011/aggregator/1.1.0" label="SDK4Mvn" packedStrategy="UNPACK_AS_SIBLING" type="R" mavenResult="true" versionFormat="MavenRelease" includeSources="true">
   <validationSets label="main">
     <contributions label="sdk">
-      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I-build"/>
+      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I202303111800"/>
```
Comment (merks) [inline publish-to-maven-central/SDK4Mvn.aggr:5]: This does not exist:

https://download.eclipse.org/eclipse/updates/4.27-I202303111800

Eventually there will be a repo like this:

https://download.eclipse.org/eclipse/updates/4.26/R-4.26-202211231800/

And soon there will be some build nested in 

https://download.eclipse.org/eclipse/updates/4.27-I-builds/

e.g.,

https://download.eclipse.org/eclipse/updates/4.27-I-builds/I20230228-0600/

that will be the declared final 4.27 build.

But we don't know that yet.

Comment (merks) [inline publish-to-maven-central/SDK4Mvn.aggr:5]: Can you please explain what you are doing and why you are doing it.  This repository does not exist:

https://download.eclipse.org/eclipse/updates/4.27-I20230311180

We don't currently know the correct repository so it cannot be changed right now.  Which is why I ask, what are you trying to do and why?

**Comment by sravanlakkimsetti** [comment]
The repositories will be created only on 11th march. This can be taken up after that

**Pull request closed without merging**